boards/arm/stm32u5a9j_dk/doc/index.rst

```diff
@@ -0,0 +1,169 @@
+|           |            | serial port-interrupt               |
++-----------+------------+-------------------------------------+
+| LPUART    | on-chip    | low power uart                      |
++-----------+------------+-------------------------------------+
+| PINMUX    | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c                                 |
++-----------+------------+-------------------------------------+
+| SPI       | on-chip    | spi                                 |
++-----------+------------+-------------------------------------+
+| FLASH     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+| DAC       | on-chip    | dac                                 |
++-----------+------------+-------------------------------------+
+| SDMMC     | on-chip    | flash memory                        |
++-----------+------------+-------------------------------------+
+
+Other hardware features have not been enabled yet for this board.
+
+The default configuration per core can be found in the defconfig file:
+``boards/arm/stm32u5a9j_dk/stm32u5a9j_dk_defconfig``
+
+Pin Mapping
+===========
+
+For mode details please refer to `STM32U5A9J-DK website`_.
+
+Default Zephyr Peripheral Mapping:
+----------------------------------
+
+- USART_1 TX/RX : PA9/PA10 (ST-Link Virtual Port Com)
+- LD3 : PE0
+- LD4 : PE1
+- User Button: PC13
+- USART_3 TX/RX : PB10/PB11
+- LPUART_1 TX/RX : PG7/PG8
+- I2C1 SCL/SDA : PG14/PG13
+- I2C2 SCL/SDA : PF1/PF0
+- I2C6 SCL/SDA : PD1/PD0
+- SPI2 SCK/MISO/MOSI/CS : PB13/PD3/PD4/PB12
+- SPI3 SCK/MISO/MOSI/CS : PG9/PG10/PG11/PG15
+- ADC1 : channel5 PA0, channel14 PC5
+- ADC2 : channel9 PA4
+- ADC4 : channel5 PF14
+
+System Clock
+============
+
+The STM32U5A9J-DK Discovery kit relies on an HSE oscillator (16 MHz crystal)
+and an LSE oscillator (32.768 kHz crystal) as clock references.
+Using the HSE (instead of HSI) is mandatory to manage the DSI interface for
+the LCD module and the USB high‑speed interface.
+
+Serial Port
+===========
+
+The STM32U5A9J Discovery kit has up to 4 USARTs, 2 UARTs, and 1 LPUART.
+The Zephyr console output is assigned to USART1 which connected to the onboard
+ST-LINK/V3.0. Virtual COM port interface. Default communication settings are
+115200 8N1.
+
+
+Programming and Debugging
+*************************
+
+See :ref:`build_an_application` for more information about application builds.
+
+
+Flashing
+========
+
+Flashing operation will depend on the target to be flashed and the SoC
+option bytes configuration.
+It is advised to use `STM32CubeProgrammer`_ to check and update option bytes
+configuration and flash the ``stm32u5a9j_dk`` target.
+
```

Comment:
Does this mean west flash can't be used ?

Reply:
Just a mistake in the doc. West flash can be used in the normal way.